rs/rosetta-api/icrc1/rosetta/runner/src/lib.rs

```diff
@@ -134,12 +132,19 @@ pub async fn start_rosetta(rosetta_bin: &Path, arguments: RosettaOptions) -> Ros
         sleep(Duration::from_millis(100)).await;
         tries_left -= 1;
     }
-    let port = maybe_port.expect("Error reading port from port file");
+    match maybe_port {
```

Comment:
Are the changes related to port file needed for the symbol check? If now, could we move them to a separate PR?

Reply:
I had the same question, otherwise the changes LGTM, thanks @NikolasHai!

Reply:
I do not really make a change to the port file. I just need the output of the error in order to be able to check the outputs content against the expected error message, as I do in the system test changes.

Reply:
Ah, I see now, thanks for clarifying!